tests/integration/test_cli_benchmark_analytics_undo_display_utils.py

```
@@ -0,0 +1,2533 @@
+        )
+        # warmup eats the only iteration → empty measured → falls back to requested_suite
+        assert eff == "io"
+
+    def test_execute_suite_iteration_success(self, tmp_path: Path) -> None:
+        from file_organizer.cli.benchmark import (
+            _execute_suite_iteration,
+            _SuiteExecutionClassification,
+            _SuiteIterationOutcome,
+        )
+
+        def fake_runner(files: list[Path]) -> _SuiteIterationOutcome:
```

Ruff (0.15.9)

[warning] Unused function argument: `files`

(ARG001)